vsphere/README.md.jinja

```diff
@@ -0,0 +1,427 @@
+{% import "macros.jinja" as macros %}
+
+{% if target == "docs" -%}
+# ![](../../_images/images-integrations/integrations-reference/vsphere/vsphere-logo.png) vSphere
+
+- [Overview](#overview)
+- [Setup](#setup)
+- [Metrics](#metrics)
+- [Features](#features)
+
+{%- endif %}
+
+## OVERVIEW
+
+### Description
+
+**This integration primarily consists of the Smart Agent monitor `vsphere`. Below is an overview of that monitor.**
+
+#### Smart Agent Monitor
+
+A VMware vSphere deployment includes physical hosts, ESXi hypervisors, virtual machines, and a vCenter Server.
+
+To report metrics for a vSphere deployment, this monitor logs into a vCenter Server and retrieves data about the deployment and its 
+real-time performance data on a regular interval.
+
+When the monitor first runs, it logs into the vCenter Server and traverses the inventory, gathering and caching all of the hosts and 
+virtual machines, and their available metrics.
+
+After this initial sweep, the monitor will query the vCenter for performance data and metrics. This query takes place every 20 seconds, 
+which is the interval at which the vCenter makes real-time performance data available. As a result, regardless of the `intervalSeconds` 
+value in the agent config, this monitor will run every 20 seconds.
+
+The monitor also refreshes, at a configurable interval, the cache of hosts, virtual machines, and metrics. By default, this refresh takes 
+place every 60 seconds; however, this interval can be changed by updating the configuration field `InventoryRefreshInterval`.
+
+#### Compatibility
+
+This monitor uses VMware’s govmomi SDK, which officially supports vCenter 6.0, 6.5, 6.7, and 7.0. While this monitor may work with 

```

Comment:
Can we go with official support for 6.5, 6.7, and 7.0 and unofficial support for 5.1, 5.5 and 6.0?

I'm using this as a reference: https://github.com/vmware/govmomi#compatibility